src/d3q27_viscoplastic/Dynamics.R

```diff
@@ -0,0 +1,86 @@
+Ave = FALSE
+x = c(0,1,-1);
+P = expand.grid(x=0:2,y=0:2,z=0:2)
+U = expand.grid(x,x,x)
+
+AddDensity(
+        name = paste("f",P$x,P$y,P$z,sep=""),
+        dx   = U[,1],
+        dy   = U[,2],
+        dz   = U[,3],
+        comment=paste("density F",1:27-1),
+        group="f"
+)
+
+# exported to VTK
+AddQuantity( name="P",unit="Pa")
+AddQuantity( name="U",unit="m/s",vector=T)
+AddQuantity( name="nu_app",unit="m2/s")
+AddQuantity( name="yield_stat")
```

Comment:
Is it possible to speficy "empty" or "-" unit? yield_stat shows if the region is yielded or not (value 0 or 1)